diff --git a/.config/awesome/rc.lua b/.config/awesome/rc.lua
index 7e93c87f6..3d77a8ebf 100644
--- a/.config/awesome/rc.lua
+++ b/.config/awesome/rc.lua
@@ -55,14 +55,14 @@ beautiful.init(gears.filesystem.get_themes_dir() .. "default/theme.lua")
 -- Default modkey (This is the super key).
 local modkey = "Mod4"
 
-beautiful.useless_gap = 3
+beautiful.useless_gap = 1
 beautiful.gap_single_client = true
 local primary_screen = screen.primary
 awful.screen.padding(primary_screen, {
-    top = 2,
-    left = 8,
-    right = 8,
-    bottom = 44
+    top = 1,
+    left = 3,
+    right = 3,
+    bottom = 33
 })
 
 -- -----------------------------------------------------------------------------
@@ -176,8 +176,8 @@ local clientkeys = gears.table.join(
             c.ontop = true
             c.floating = true
 
-            c.width = 556
-            c.height = 312
+            c.width = 900
+            c.height = 600
             awful.placement.bottom_right(c, {
                 honor_workarea = true,
                 margins = { right = 16, bottom = 77 }
diff --git a/.config/polybar/config.ini b/.config/polybar/config.ini
index cfb3332fa..4ede30fdc 100644
--- a/.config/polybar/config.ini
+++ b/.config/polybar/config.ini
@@ -7,7 +7,7 @@ include-file=~/.config/polybar/system.ini
 [bar-base]
 type = bar
 width = 100%
-height = 1.8%
+height = 1.5%
 radius = 0
 dpi-x = 96
 dpi-y = 96
@@ -29,17 +29,17 @@ override-redirect = true
 fixed-center = true
 
 ; Main.
-font-0 = JetBrainsMono Nerd Font:pixelsize=13:weight=bold;4
+font-0 = JetBrainsMono Nerd Font:pixelsize=10:weight=bold;3
 ; Main half circle separators.
-font-1 = JetBrainsMono Nerd Font:pixelsize=19;4
+font-1 = JetBrainsMono Nerd Font:pixelsize=11;3
 ; Logo
-font-2 = JetBrainsMono Nerd Font:pixelsize=23:weight=bold;5
+font-2 = JetBrainsMono Nerd Font:pixelsize=18:weight=bold;5
 ; Spaces.
-font-3 = JetBrainsMono Nerd Font:pixelsize=6
+font-3 = JetBrainsMono Nerd Font:pixelsize=4
 ; For mod sep.
-font-4 = JetBrainsMono Nerd Font:pixelsize=15:antialias=false;4
+font-4 = JetBrainsMono Nerd Font:pixelsize=13:antialias=false;4
 ; For workspaces
-font-5 = JetBrainsMono Nerd Font:pixelsize=17:antialias=true;5
+font-5 = JetBrainsMono Nerd Font:pixelsize=13:antialias=true;3
 
 [bar/top]
 inherit = bar-base
diff --git a/.config/polybar/system.ini b/.config/polybar/system.ini
index 35ab9f82b..9e3b21365 100644
--- a/.config/polybar/system.ini
+++ b/.config/polybar/system.ini
@@ -127,7 +127,7 @@ format-background = ${colors.background}
 tray-background = ${colors.background}
 format-padding = 0
 tray-padding = 0
-tray-spacing = 6
+tray-spacing = 2
 tray-size = 21
 
 [module/gitlab-notifications]
diff --git a/.config/rofi/config.rasi b/.config/rofi/config.rasi
index 9aa8dc9bb..8eef931cf 100644
--- a/.config/rofi/config.rasi
+++ b/.config/rofi/config.rasi
@@ -37,7 +37,7 @@
 
 configuration {
     show-icons:                true;
-    font:                      "JetBrainsMono Nerd Font Bold 13";
+    font:                      "JetBrainsMono Nerd Font Bold 12";
     icon-theme:                "Papirus-Dark";
     drun {
         fallback-icon:          "kblackbox";
@@ -51,7 +51,7 @@ configuration {
 // Main window.
 window {
     width:                      35%;
-    height:                     41.1112%;
+    height:                     42.593%;
     border-radius:              @outer-border-radius;
     border-color:               @border;
     background-color:           @background;
diff --git a/.config/wezterm/wezterm.lua b/.config/wezterm/wezterm.lua
index 0806b8245..1716f8a2d 100644
--- a/.config/wezterm/wezterm.lua
+++ b/.config/wezterm/wezterm.lua
@@ -5,16 +5,16 @@ local config = {}
 config.default_prog = { 'fish' }
 config.enable_tab_bar = false
 
-config.max_fps = 100
-config.animation_fps = 100
+config.max_fps = 75
+config.animation_fps = 75
 
 -- config.window_background_opacity = 0.94
 config.window_background_opacity = 1
-config.use_resize_increments = true
+config.use_resize_increments = false
 
-config.font_size = 13
+config.font_size = 11.3
 config.underline_thickness = 1
-config.underline_position = -5.0
+config.underline_position = -4.0
 -- config.freetype_load_target = "Normal"
 config.bold_brightens_ansi_colors = false
 -- config.freetype_render_target = 'Normal'
diff --git a/.scripts/startup.sh b/.scripts/startup.sh
index 6ff0408c0..33926d3ed 100755
--- a/.scripts/startup.sh
+++ b/.scripts/startup.sh
@@ -6,7 +6,7 @@ pkill polybar
 
 # Core components (order is important!)
 (
-    ~/.scripts/screenlayout/single_1440p.sh
+    ~/.scripts/screenlayout/single_1080p.sh
     
     # Check if wallpapers directory exists
     if [ ! -d ~/.private/wallpapers ]; then
